modules/customization/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -0,0 +1,116 @@
+[id='proc-customize-rhdh-sidebar-menuitems_{context}']
+= Customizing the sidebar navigation menu items for your {product-short} instance
+
+The sidebar menu in {product} consists of two main parts:
+
+* *Main menu items*: These items are the static menu items that form the core navigation structure of sidebar. These menu items remain consistent and are predefined.
+
+* *Dynamic plugin menu items*: These items are displayed beneath the main menu and can be customized based on the plugins installed. The main menu items section is dynamic and can change based on your preferences and installed plugins.
```

Comment:
```suggestion
* *Dynamic plugin menu items*: These items are displayed beneath the main menu and can be customized based on the plugins installed. The dynamic plugin menu items section is dynamic and can change based on your preferences and installed plugins.
```

Reply:
@ciiay, the sentence states 'dynamic plugin menu items section is dynamic,' which seems redundant. Adding 'dynamic' as a prefix doesn't feel necessary, so I would suggest keeping it as is.